sdk/core/System.ClientModel/README.md

```diff
@@ -25,67 +24,144 @@ None needed for `System.ClientModel`.
 
 ### Authenticate the client
 
-The `System.ClientModel` package provides a `KeyCredential` type for authentication.
+The `System.ClientModel` package provides an `ApiKeyCredential` type for authentication.
 
 ## Key concepts
 
-The main shared concepts of `System.ClientModel` include:
+The main concepts in `System.ClientModel` include:
 
 - Configuring service clients (`ClientPipelineOptions`).
 - Accessing HTTP response details (`ClientResult`, `ClientResult<T>`).
-- Exceptions for reporting errors from service requests in a consistent fashion (`ClientResultException`).
-- Customizing requests (`RequestOptions`).
-- Providing APIs to read and write models in different formats (`ModelReaderWriter`).
+- Handling exceptions that result from failed requests (`ClientResultException`).
+- Customizing HTTP requests (`RequestOptions`).
+- Reading and writing models in different formats (`ModelReaderWriter`).
+
+Below, you will find sections explaining these shared concepts in more detail.
 
 ## Examples
```

Comment:
I think the first sample should show a very simple client implementation with one service method.

Reply:
Ah, cool. I had one but I had thought we wanted these just for end-users so I took it out.  I can dig it back up!

Reply:
Updated.  Let me know if you had something different in mind!